apps/emqx_resource/src/emqx_resource_manager.erl

```diff
@@ -459,7 +464,7 @@ handle_event(
 handle_event(
     {call, From}, {remove_channel, ChannelId}, _State, Data
 ) ->
-    handle_not_connected_remove_channel(From, ChannelId, Data);
+    handle_not_connected_or_connecting_remove_channel(From, ChannelId, Data);
```

Comment:
Since there's a clause for the `connecting` state above this that doesn't use this function:

```suggestion
    handle_not_connected_remove_channel(From, ChannelId, Data);
```
?

Reply:
Thanks! Changed the name to something that makes more sense: [fe6bf9d](https://github.com/emqx/emqx/pull/11857/commits/fe6bf9d129f36e8d8f486f84ad969f9b24e9e094)